### PR TITLE
add information about block retrieval

### PR DESCRIPTION
**Pull request by paulperegud**

### CODE REVIEW

File: plasma_framework/docs/integration-docs/integration-doc.md [hunk 1]
```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, compute block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, learn their commited hash values by reading `blocks` mapping.
```
Comment (pdobacz) [inline plasma_framework/docs/integration-docs/integration-doc.md]: a couple of points:
 - how to get `childBlockInterval` and `nextChildBlock`
 - `commited hash` -> submitted root of the block Merkle tree (consistent with `# Block submission`)
 - `reading...mapping` -> I think this deserves listing out as a contract call, similar to [this for example](https://github.com/omisego/plasma-contracts/blob/46ccbd8f3a544c94955dd633d2c8ee7565e2bbbd/plasma_framework/docs/integration-docs/integration-doc.md#challenging-in-flight-transaction-output-exit)

File: plasma_framework/docs/integration-docs/integration-doc.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, read the values of `nextChildBlock` and `childBlockInterval` from the contract (e.g. via [`eth_call`](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_call)). Compute block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, read values of roots of Merkle trees of blocks by calling `PlasmaFramework.blocks(blknum)`.
```
Comment (pdobacz) [inline plasma_framework/docs/integration-docs/integration-doc.md]: `(e.g. via [`eth_call`](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_call)`

hm, not the thing that is needed here to answer this:
> how to get childBlockInterval and nextChildBlock

Not asking for general ethereum-integration advice, but on the level of the contract code and defining the values you're using. What API one needs to hit, which contract? If applicable, something along the lines of `PlasmaFramework.blocks(blknum)` you did.

Please be mindful about that your `nextChildBlock` and `childBlockInterval` only appear in your added section. A reader, who doesn't know what those are, will not know how to "integrate".

File: plasma_framework/docs/integration-docs/integration-doc.md [hunk 3]
```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, read `PlasmaFramework.nextChildBlock()` and `PlasmaFramework.childBlockInterval()`. Compute block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, read values of roots of Merkle trees of blocks by calling `PlasmaFramework.blocks(blknum)`.
```
Comment (kevsul) [inline plasma_framework/docs/integration-docs/integration-doc.md]:
```suggestion
To learn what blocks need to be retrieved during sync, read `PlasmaFramework.nextChildBlock()` and `PlasmaFramework.childBlockInterval()`. Compute the block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, read the Merkle tree root of each block by calling `PlasmaFramework.blocks(blknum)`.
```

**Comment by kevsul** [comment]
Sorry , I had missed this originally.
LGTM, some minor rewording for clarification

**Comment by paulperegud** [comment]
Committed Kevin's suggestion. Please re-review. Need two confirmations to merge.

File: plasma_framework/docs/integration-docs/integration-doc.md [hunk 4]
```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, read `PlasmaFramework.nextChildBlock()` and `PlasmaFramework.childBlockInterval()`. Compute the block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, read the Merkle tree root of each block by calling `PlasmaFramework.blocks(blknum)`.
```
Comment (pnowosie) [inline plasma_framework/docs/integration-docs/integration-doc.md]: `...Compute the block numbers by iterating` isn't very clear for me. I guess we need to put some math at it.
Block numbers are elements of arithmetic progression with a common difference of `childBlockInterval`, starting from ...

**Pull request merged**